src/axolotl/monkeypatch/fastchat_conversation_turns.py

```diff
@@ -123,6 +123,14 @@ def get_turns(  # pylint: disable=too-many-return-statements
             else:
                 yield role, ""
         return
+    if self.sep_style == SeparatorStyle.GEMMA:
+        if self.system_message:
+            raise ValueError("Gemma chat template does not support system messages")
+        for i, (role, message) in enumerate(self.messages):
+            prefix = "<bos>" if i == 0 else ""
```

Comment:
I am a bit confused about this. I don't think you need to add `bos` manually like this. Axolotl should prepend it.

Reply:
since this is Gemma specific, and this function doesn't have access to the tokenizer, I think this is fine for now. in other formats in this function, we have literal "<s>" strings.